examples/abitest/client/cpp/httptest.cc

```diff
@@ -19,15 +19,19 @@
 #include "absl/strings/escaping.h"
 #include "glog/logging.h"
 #include "httplib_config.h"
+#include "oak_abi/proto/identity.pb.h"
 #include "oak_abi/proto/label.pb.h"
 
 const char* CA_CERT_PATH = "../../../../../../../../../examples/certs/local/ca.pem";
 const int PORT = 8383;
+const char* SIGNED_HASH_BASE64 =
```

Comment:
Perhaps add a comment on where these come from / how they can be generated here as well?